diff --git a/dist/cjs/node/vite/index.js b/dist/cjs/node/vite/index.js
index 7407390885ee7b990c1b4319f0e976209109630b..e7cd38bb4fcd6f34847062e6b050e47c43242bfd 100644
--- a/dist/cjs/node/vite/index.js
+++ b/dist/cjs/node/vite/index.js
@@ -23,7 +23,7 @@ function plugin(config) {
         (0, buildConfig_1.buildConfig)(),
         (0, retrieveDevServer_1.retrieveDevServer)(),
         (0, packageJsonFile_1.packageJsonFile)(),
-        ...(0, importBuild_1.importBuild)(),
+        // ...(0, importBuild_1.importBuild)(),
         (0, previewConfig_1.previewConfig)(),
         (0, manifest_1.manifest)(config),
         (0, printShieldGenResult_1.printShieldGenResult)()