diff --git a/dist/cjs/node/vite/index.js b/dist/cjs/node/vite/index.js
index 7407390885ee7b990c1b4319f0e976209109630b..fe054d7a87a6c3f0d50d3a526d56b29548bb6486 100644
--- a/dist/cjs/node/vite/index.js
+++ b/dist/cjs/node/vite/index.js
@@ -20,10 +20,10 @@ function plugin(config) {
         (0, transform_1.transform)(),
         (0, commonConfig_1.commonConfig)(),
         ...(0, devConfig_1.devConfig)(),
-        (0, buildConfig_1.buildConfig)(),
+        // (0, buildConfig_1.buildConfig)(),
         (0, retrieveDevServer_1.retrieveDevServer)(),
         (0, packageJsonFile_1.packageJsonFile)(),
-        ...(0, importBuild_1.importBuild)(),
+        // ...(0, importBuild_1.importBuild)(),
         (0, previewConfig_1.previewConfig)(),
         (0, manifest_1.manifest)(config),
         (0, printShieldGenResult_1.printShieldGenResult)()